pkgs/by-name/du/duckstation/package.nix

```diff
@@ -1,35 +1,43 @@
 { lib
-, stdenv
-, fetchFromGitHub
```

Comment:
Unnecessary change - moving inputs enlarges the diff for no good reason.

Reply:
> Unnecessary change - moving inputs enlarges the diff for no good reason.

Still unresolved.

Reply:
Sorting them seems reasonable to me.

Reply:
> Sorting them seems reasonable to me.

It does seem stupid to doronbehar, let's solve this.